torch/_subclasses/functional_tensor.py

```diff
@@ -342,6 +342,23 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         if kwargs is None:
             kwargs = {}
 
+        if self.export:
```

Comment:
mentioned to @tugsbayasgalan offline, but I would much rather have this logic be populated into export's decomposition table than have it get hardcoded directly into `FunctionalTensorMode`.